2020-01-29-governance-wg-meeting.md

```diff
@@ -0,0 +1,170 @@
+---
+breaks:  false # See: https://github.com/hackmdio/codimd/issues/40#issuecomment-172927690
+private: false # See: https://github.com/hyphacoop/hyphacoop-chatbot#archive
+---
+# 2020-01-29 Hypha Worker Co-op: Governance WG Meeting
+
+<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
+
+Time: Wednesday, Jan 29, 2020 @ 10am-12pm  
+Location: https://jitsi.hypha.coop/hyphacoop  
+Attending: dc, pc  
+Cultivator: dc  
+Notetaker: dc, pc  
+
+## Agenda
+
+| Item                                        | Time (min) |
+|:--------------------------------------------|-----------:|
+| [Personal Checkins](#Personal-Checkins)     |          5 |
+| [Task Board Review](#Task-Board-Review)     |         20 |
+| [Discussions](#Discussions)                 |         40 |
+| [Co-work](#Co-work        )                 |         50 |
+| [Process Checkout](#Process-Checkout)       |          5 |
+
+## Notes
+
+### Personal Checkins
+
+- How was last night? 
+    - pc: Civic Tech Saint John, was great! Felt nice to offer something
+    - dc: Civic Tech Toronto: nice to revisit the community :)
+
+
+### Task Board Review
+
+- [Governance][gov-wg] :white_check_mark: [`wg:governance`][l-gov]
+    - Archive _Done_ tasks :tada:
+        - done.
+    - Review [HIGH `[priority-★★★]`][l-pri-hi]
+        - #155 dc did first pass. reviewed some gov and OCA docs.
+        - #101 bylaws
+            - what's in scope of bylaws?
+            - dc: story of student union: template bylaws and added extra. operating book had bylaw-y comments. split was all off.
+            - dc: plain language
+            - pc: would love to read "bad bylaws" to develop stronger sense
+            - dc: good chapters in "ours to hack and own" platform coop book
+        - #125 WSIB
+            - dc: can get tender letter on file. some clients might want
+            - WSIB not same as meeting OSHA (and this is operations)
+            - dc: WSIB primarily protects employers (cynical take). employees can still file for claims if we don't pay in
+            - dc: we should think of this as an "employer protection" task, not "employee protection"
```

Comment:
@YurkoWasHere @ASoTNetworks FYI. I forgot this aspect, but should have had it in mind while we were practicing developing a more clear stance on issues in our infra WG meeting (re: WSIB). While our rationale as individuals might be less valid, our stance as WG members still seems strong, despite the misunderstanding